src/plugins/telemetryTable/components/TelemetryFilterIndicator.vue

```diff
@@ -128,15 +134,38 @@
                 this.setFilterNames();
                 this.checkFiltersForMixedValues();
                 this.setLabels();
+            },
+            addChildren(child) {
+                let keyString = this.openmct.objects.makeKeyString(child.identifier);
+                this.domainObjectsKeyString[keyString] = '';
+                this.checkFiltersForMixedValues();
+                this.setLabels();
+            },
+            removeChildren(identifier) {
+                let keyString = this.openmct.objects.makeKeyString(identifier);
+                delete this.domainObjectsKeyString[keyString];
             }
         },
         mounted() {
             let filters = this.table.configuration.getConfiguration().filters || {};
+            this.domainObjectsKeyString = {};
```

Comment:
What is this object for? I only see it being set to '' or deleted?

Reply:
@akhenry `domainObjectsKeyString` is key'd on telemetry object ids that are in the composition. It's used to check against `telemetryFilters` which is key'd on only telemetry object ids that have a filter value set (at line 117).

Reply:
@psarram Its attributes are only ever set to '', or deleted though? This is what I'm having trouble with.

Reply:
@akhenry That's a fair point. It's not a proper data structure in this case. I changed the object to a Set to store only the keyStrings.

Reply:
It seems to me as though it's counting the objects in the composition? Wouldn't an integer do for this purpose?

Reply:
@akhenry It's keeping the unique key string of objects in the composition. Integer won't work as it should be known which key string to update when an object is removed or added to the composition.